<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>244</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="But_Divide">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>100</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255,151,57);
	color:white;
	border: 1px solid gray;
}

QPushButton:pressed {
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1,
								stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_Percent">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>100</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(215,215,215);
	border: 1px solid gray;
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1,
								stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_PlusMinus">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>100</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(215,215,215);
	border: 1px solid gray;
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1,
								stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButAC">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(215,215,215);
	border: 1px solid gray;
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1,
								stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>ะะก</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_multi">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>161</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255,151,57);
	color:white;
	border: 1px solid gray;
}

QPushButton:pressed {
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1,
								stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_Minus">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>222</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255,151,57);
	color:white;
	border: 1px solid gray;
}

QPushButton:pressed {
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1,
								stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_Plus">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>283</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255,151,57);
	color:white;
	border: 1px solid gray;
}

QPushButton:pressed {
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1,
								stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButRavno">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>344</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255,151,57);
	color:white;
	border: 1px solid gray;
}

QPushButton:pressed {
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1,
								stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_Dot">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>344</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(215,215,215);
	border: 1px solid gray;
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0,y1:0,x2:0,y2:1,
								stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_0">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>344</y>
      <width>122</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>122</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid gray;
}

QPushButton:pressed {
	background-color:qlineargradient(x1:0,y1:0,x2:0,y2:1,
								stop: 0 #dadbde, stop: 1 #f6f7fa)
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_6">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>222</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid gray;
}

QPushButton:pressed {
	background-color:qlineargradient(x1:0,y1:0,x2:0,y2:1,
								stop: 0 #dadbde, stop: 1 #f6f7fa)
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_7">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>161</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>21</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid gray;
}

QPushButton:pressed {
	background-color:qlineargradient(x1:0,y1:0,x2:0,y2:1,
								stop: 0 #dadbde, stop: 1 #f6f7fa)
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_2">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>283</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid gray;
}

QPushButton:pressed {
	background-color:qlineargradient(x1:0,y1:0,x2:0,y2:1,
								stop: 0 #dadbde, stop: 1 #f6f7fa)
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_5">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>222</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid gray;
}

QPushButton:pressed {
	background-color:qlineargradient(x1:0,y1:0,x2:0,y2:1,
								stop: 0 #dadbde, stop: 1 #f6f7fa)
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_9">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>161</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid gray;
}

QPushButton:pressed {
	background-color:qlineargradient(x1:0,y1:0,x2:0,y2:1,
								stop: 0 #dadbde, stop: 1 #f6f7fa)
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>283</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid gray;
}

QPushButton:pressed {
	background-color:qlineargradient(x1:0,y1:0,x2:0,y2:1,
								stop: 0 #dadbde, stop: 1 #f6f7fa)
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>222</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid gray;
}

QPushButton:pressed {
	background-color:qlineargradient(x1:0,y1:0,x2:0,y2:1,
								stop: 0 #dadbde, stop: 1 #f6f7fa)
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_8">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>161</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid gray;
}

QPushButton:pressed {
	background-color:qlineargradient(x1:0,y1:0,x2:0,y2:1,
								stop: 0 #dadbde, stop: 1 #f6f7fa)
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="But_3">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>283</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border: 1px solid gray;
}

QPushButton:pressed {
	background-color:qlineargradient(x1:0,y1:0,x2:0,y2:1,
								stop: 0 #dadbde, stop: 1 #f6f7fa)
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLabel" name="result_show">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>240</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>240</width>
      <height>71</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{ 
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border-bottom: 1px solid gray;
}

background-color : white;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
